src/index.js

```diff
@@ -32,14 +32,15 @@ async function run() {
   }
 
   // Monkey Patch the superagent for proxy use
-  const proxy = args.proxy_url;
+  const proxy = params.proxy_url;
   if (proxy) {
+    log.info(`PROXY: ${proxy}`)
```

Comment:
Linting failed on this line. Please remove it and we can merge.

```
/home/circleci/repo/src/index.js
  37:32  error  Missing semicolon  semi
```

Reply:
Hi @fyockm, stuff got busy for me and I got dragged away from this. I've made the change 👍